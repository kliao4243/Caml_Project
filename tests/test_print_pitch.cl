/* author: Yuanji Huang */
int main(){
    print(4^4);
    print(5#4);
    print(4b4);
}
