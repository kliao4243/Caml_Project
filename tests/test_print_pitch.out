4^4
5#4
4b4